src/babashka/process.cljc

```diff
@@ -273,6 +274,8 @@
            cmd
            (build cmd opts))
          cmd (vec (.command pb))
+         interceptor-map {:cmd cmd}
```

Comment:
I get the map creation can be pushed inside the when expression, to only do that when it's necessary

Reply:
that's a great shout - I was in the mindset of "we might need to re-use the map later if we end up with a post-fn or something later on", but we could still do that in a when, and it can be separate when it needs to be separate